src/SourceGenerators/XamlGenerationTests/XamlGenerationTests.csproj

```diff
@@ -15,6 +15,7 @@
 	<ItemGroup>
 		<PackageReference Include="Uno.SourceGenerationTasks" />
 		<PackageReference Include="Uno.Core.Build" />
+		<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls" />
```

Comment:
It'd be best to avoid external dependencies here (those depend on Uno, which is defined inside the currently building solution). Try recreating the defined type instead.